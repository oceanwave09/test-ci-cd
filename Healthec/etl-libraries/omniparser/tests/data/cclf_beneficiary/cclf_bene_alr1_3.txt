BENE_MBI_ID,BENE_HIC_NUM,BENE_1ST_NAME,BENE_LAST_NAME,BENE_SEX_CD,BENE_BRTH_DT,BENE_DEATH_DT,MASTER_ID,REV_LINE_CNT
1AAAAMKR10,,2501FirstName,2501LastName,2,01/01/1965,,202084193,16
1AAAAMKR20,,2502FirstName,2502LastName,2,01/01/1947,,383593303,2
