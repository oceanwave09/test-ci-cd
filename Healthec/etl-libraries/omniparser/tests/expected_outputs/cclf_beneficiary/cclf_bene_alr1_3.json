[
	{
		"bene_1st_name": "2501FirstName",
		"bene_brth_dt": "01/01/1965",
		"bene_last_name": "2501LastName",
		"bene_mbi_id": "1AAAAMKR10",
		"bene_sex_cd": "2",
		"master_id": "202084193",
		"rev_line_cnt": "16",
		"row_id": "23df2245-dc9d-4365-b44a-3359d44c3db8"
	},
	{
		"bene_1st_name": "2502FirstName",
		"bene_brth_dt": "01/01/1947",
		"bene_last_name": "2502LastName",
		"bene_mbi_id": "1AAAAMKR20",
		"bene_sex_cd": "2",
		"master_id": "383593303",
		"rev_line_cnt": "2",
		"row_id": "f6c7d13e-0a98-4468-84c5-e4e1f13a34eb"
	}
]
